docs/spark.rst

```diff
@@ -96,7 +96,32 @@ logging (for Tensorboard) using the Estimator ``Store`` abstraction.  Stores are
 artifacts including intermediate representations of the training data.  Horovod natively supports stores for HDFS
 and local filesystems.
 
-`Petastorm <https://github.com/uber/petastorm/blob/master/petastorm/pytorch.py#L258>` based data loader is used by default, but user can define a custom data loader by override the `BaseDataLoader` interface. A async data loader mixin can also added on top of the data loader.
+`Petastorm <https://github.com/uber/petastorm/blob/master/petastorm/pytorch.py#L259>`__ based data loader is used by default,
+but user can define a custom data loader by overriding the `BaseDataLoader` interface. An async data loader mixin can also
+be added on top of the data loader.  Additionally, the KerasEstimator supports a DataModule argument, similar
+to the Lightning DataModule, which abstracts the data loading and allows for alternative implementations.  For example,
+the NVTabularDataModule integrates the `KerasSequenceLoader <https://github.com/NVIDIA-Merlin/NVTabular/blob/main/nvtabular/loader/tensorflow.py>`__
+from NVTabular to enable GPU-accelerated data loading.  Note, however, due to the complexity installation, NVTabular
+recommends the use of a `conda` environment or a pre-built docker image.
```

Comment:
You can also mention `docker/horovod-nvtabular/Dockerfile` here as an example to install Horovod with NVTabular.

Reply:
Good point, done.